java/util/text.mcdoc

```diff
@@ -209,10 +209,12 @@ dispatch minecraft:click_event[show_dialog] to struct ShowDialog {
 dispatch minecraft:click_event[custom] to struct CustomAction {
 	/// ID of a custom action.
 	/// Has no functionality on vanilla servers.
-	id: #[id] string,
-	payload?: string,
+	id: #[id] #[dispatcher_key="mcdoc:custom_payload"] string,
+	payload?: mcdoc:custom_payload[[%key]],
```

Comment:
`%key` here is always `payload`. if anything it should be `mcdoc:custom_payload[[id]]`

But since we can't do type-checking of the dialog inputs for the `dynamic/custom` dialog action (and additions), maybe just `any` would work maybe work better here.